lib/services/ios-debug-service.ts

```diff
@@ -205,9 +208,20 @@ class IOSDebugService implements IDebugService {
     private openDebuggingClient(): IFuture<void> {
         return (() => {
             let inspectorPath = this.getInspectorPath().wait();
-            let inspectorApplicationPath = path.join(inspectorPath, "NativeScript Inspector.app");
-            let inspectorSourceLocation = path.join(inspectorPath, "Safari/Main.html");
-            let cmd = `open -a '${inspectorApplicationPath}' --args '${inspectorSourceLocation}' '${this.$projectData.projectName}'`;
+            let cmd: string = null;
+            
+            this.$projectDataService.initialize(this.$projectData.projectDir);
+            let platformData = this.$platformsData.getPlatformData(this.platform);
+            let frameworkVersion = this.$projectDataService.getValue(platformData.frameworkPackageName).wait().version;
+            if(semver.lt(frameworkVersion, "1.2.0")) {
+                let safariPath = path.join(inspectorPath, "Safari/Main.html");
+                cmd = "open -a Safari " + safariPath;                
+            } else {
+                let inspectorApplicationPath = path.join(inspectorPath, "NativeScript Inspector.app");
+                let inspectorSourceLocation = path.join(inspectorPath, "Safari/Main.html");
```

Comment:
this path is calculated both in if and else, consider moving it out and calculating it once.